### PR TITLE
Allow the role to work with user and password instead of token

### PR DESCRIPTION
**Pull request by craigbr**
Change the keystone_url to v2 if a v3 one is given

### CODE REVIEW

**Comment by tkuhlman** [comment]
Please update the README also.

Regarding the cacert is that needed? With the monasca api endpoint when the url was ssl we just assumed a valid CA was loaded on the machine running the script. We could do the same here.

File: tasks/main.yml [hunk 1]
```diff
@@ -2,30 +2,17 @@
...
-- name: Keystone User - Add role(s)
-  keystone_user:
-    user={{ item.key }}
-    role={{ item.value.role | default("") }}
-    tenant={{ item.value.project }}
-    endpoint={{ keystone_url }}
-    token={{ keystone_admin_token }}
-  with_dict: keystone_users
+- name: Copy cacert file if given
+  copy: src={{ keystone_cacert }} dest={{ keystone_local_cacert }} mode=0744
```
Comment (tkuhlman) [inline tasks/main.yml]: 644

**Comment by tkuhlman** [comment]
Looks good except that minor 744 mode on the file. We should simplify the monasca-vagrant and installer config to just feed in a v3 url rather than the v2 url. If you like I can do that.

**Comment by craigbr** [comment]
I fixed the file permission. I already have the monasca-vagrant change ready so I can do it and the installer config.
craig

**Pull request merged**